portfolio/src/main/webapp/script.js

```diff
@@ -30,21 +30,21 @@ function addRandomGreeting() {
 }
 
 function composeWorld(f) {
```

Comment:
Add function comments (what the function does + description of what the expected parameter is*)

*In cases where the parameter is something like "name" it's enough to have a quick 
@param {string} name 
In cases where the parameter name is a bit less clear, like this "f" parameter, it might help to have 
@param {<type} f  // Comment about what f is expected to be